.github/rpm/twilio.spec

```diff
@@ -0,0 +1,50 @@
+# Spec file is needed to build a rpm package
+Name:           twilio
+Version:        %{_version}
+Release:        1%{?dist}
```

Comment:
[Q] What does `?` indicate? Do we define `dist` anywhere like the `_version` in action variables?